docs/docsite/rst/reference_appendices/test_strategies.rst

```diff
@@ -188,6 +188,8 @@ This is the great culmination of embedded tests:
           ansible.builtin.command: /usr/bin/add_back_to_pool {{ inventory_hostname }}
           delegate_to: 127.0.0.1
 
+If you need a handler to run before a particular role, use the ``meta`` module with the ``flush_handlers`` option. Consider placing this task in your test role itself. For more information, see :ref:`handlers`.
```

Comment:
This seems to be out of context. We do document the `flush_handlers` meta task in the appropriate sections.

Reply:
We do mention them here if that helps - https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html#controlling-when-handlers-run

Reply:
okay rereading this and the issue that spawned this PR.  Seems I put the wrong info in the issue. I think this new line likely belongs at https://github.com/ansible/ansible/blame/devel/docs/docsite/rst/reference_appendices/test_strategies.rst#L194 instead. What do you think @mkrizek ?

Reply:
I think @samccann meant this https://docs.ansible.com/ansible/latest/collections/ansible/builtin/meta_module.html

@mkrizek, do you think we should remove the paragraph you highlighted and instead put there a reference to ... https://docs.ansible.com/ansible/latest/collections/ansible/builtin/meta_module.html    ?

Thanks

Reply:
I wonder if it would be better to just fix the example playbook rather than mention what could go wrong with it?
We could rewrite the playbook and move a "call" to `apply_testing_checks` role into `post_tasks` via `include_role` (since `flush_handlers` is implicitly called before `post_tasks` is executed). Or run `flush_handlers` explicitly to make the intent more obvious.

What do you think?

Reply:
_I wonder if it would be better to just fix the example playbook rather..._ you mean the play/playbook that immediately precedes?

`    ---
    - hosts: webservers
      serial: 5
      pre_tasks:
        - name: take out of load balancer pool
          ansible.builtin.command: /usr/bin/take_out_of_pool {{ inventory_hostname }}
          delegate_to: 127.0.0.1
      roles:
         - common
         - webserver
         - apply_testing_checks
      post_tasks:
        - name: add back to load balancer pool
          ansible.builtin.command: /usr/bin/add_back_to_pool {{ inventory_hostname }}
          delegate_to: 127.0.0.1
`

Reply:
+1 for updating the example